ENIGMAsystem/SHELL/Audio_Systems/DirectSound/DSsystem.h

```diff
@@ -34,13 +39,13 @@ int get_free_channel(double priority);
 #ifdef DEBUG_MODE
 #define get_sound(snd, id, failure)                                              \
   if (id < 0 or size_t(id) >= sound_resources.size() or !sound_resources[id]) {  \
-    show_error("Sound " + enigma_user::toString(id) + " does not exist", false); \
+    enigma_user::show_error("Sound " + enigma_user::toString(id) + " does not exist", false); \
```

Comment:
Oh, also, you kind of broke the formatting, here.

Reply:
Ok: b60815f9f89b67d8502ccbe7d4b2c58d7189cc9f

Yeah, I knew I missed a few of these.